app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,20 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self, nickname: str,
+            musical_instrument: str,
+            bow_level: int
+    ) -> None:
```

Comment:
Code Efficiency: You should use type hinting for your methods and functions. It's a good practice to use it, as it makes your code more readable and easier to understand. For example, instead of `def __init__(self, nickname, musical_instrument, bow_level):`, you should use `def __init__(self, nickname: str, musical_instrument: str, bow_level: int):`.